### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][ColourContrast][MAS1.4.3][IntelliJ][Wipro]: Colour contrast ratio is below 4.5:1 for selected created "Webapp" and "Build" under Before launch expand collapse button in Run\debug configurations window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1385]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select "edit configurations" present at right side of the dialog.
4. "Run/Debug Configurations" dialog will open.
5. In "Run/Debug Configurations" dialog navigate and select "Use existing web app\create web app" radio button.
5. Tab to "Web app" and "Artifact"  under Before launch expand collapse button and  observe the colour contrast ratio.

**Expected Result:**
Colour contrast ratio should be greater than or equal to 4.5:1 for  selected created "Web app" and "Build"  under Before launch expand collapse button in Run\debug configurations window.

**Actual Result:**
 Colour contrast ratio is  4.4:1 for selected created "Web app" and "Build"  under Before launch expand collapse button in Run\debug configurations window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version**: Version 1709(Os Build 16299.192)
**CCA version**: 2.5.0.0

![cca for created webapp and build in rundebug config](https://user-images.githubusercontent.com/22956130/35681851-9dead142-0713-11e8-8d1e-8caabf1779c5.png)

**Comment by Eskibear** [comment]
It has external dependency, IntelliJ IDEA handles its own theme, refer to:
https://youtrack.jetbrains.com/issue/IDEA-132299

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749852
